modules/antismash.nf

```diff
@@ -0,0 +1,51 @@
+process ANTISMASH {
+
+    tag "${cluster_name}"
+    
+    publishDir(
+        path: "${params.outdir}",
+        saveAs: {
+            filename -> {
+                String cluster_prefix = cluster_name.substring(0, cluster_name.length() - 2);
+                return "species_catalogue/${cluster_prefix}/${cluster_name}/genome/${cluster_name}_antismash.gff"
```

Comment:
This has been properly fixed in https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/pull/109. Rather than including full antiSMASH output, the GFF now contains all of the relevant information.